### PR TITLE
Feat/poc mongodb plugin

### PR DESCRIPTION
**Pull request by morhag90**
Close: #1959 

## PR Details

Add support to mongoDB prisma creation. 
## PR Checklist

- [x] Tests for the changes have been added
- [x] `npm test` doesn't throw any error

### CODE REVIEW

File: helm/charts/dsg-runner-infra/templates/sensor.yaml [hunk 1]
```diff
@@ -31,7 +31,7 @@ spec:
                     restartPolicy: Never
                     containers:
                       - name: dsg-container
-                        image: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator:sha-27613bb"
+                        image: "407256539111.dkr.ecr.us-east-1.amazonaws.com/amplication-data-service-generator:feat/poc-mongodb-plugin"
```
Comment (overbit) [inline helm/charts/dsg-runner-infra/templates/sensor.yaml:34]: It needs update before we can merge it.

File: ee/packages/amplication-git-pull-service/scripts/clean.ts [hunk 2]
```diff
@@ -40,8 +40,7 @@ async function dropTypes(prisma: PrismaClient, types: string[]) {
 async function getTables(prisma: PrismaClient): Promise<string[]> {
   const results: Array<{
     tablename: string;
-  }> =
-    await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
+  }> = await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
```
Comment (overbit) [inline ee/packages/amplication-git-pull-service/scripts/clean.ts:43]: Undo this in order to fix the ci pipeline. 
Locally you'd need to remove your node_modules and run `npm i`

File: packages/amplication-data-service-generator/src/server/static/scripts/clean.ts [hunk 3]
```diff
@@ -40,8 +40,7 @@ async function dropTypes(prisma: PrismaClient, types: string[]) {
 async function getTables(prisma: PrismaClient): Promise<string[]> {
   const results: Array<{
     tablename: string;
-  }> =
-    await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
+  }> = await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
```
Comment (overbit) [inline packages/amplication-data-service-generator/src/server/static/scripts/clean.ts:43]: Undo this in order to fix the ci pipeline.
Locally you'd need to remove your node_modules and run npm i

File: packages/amplication-prisma-db/prisma/clean.ts [hunk 4]
```diff
@@ -38,8 +38,7 @@ async function dropTypes(prisma: PrismaClient, types: string[]) {
 async function getTables(prisma: PrismaClient): Promise<string[]> {
   const results: Array<{
     tablename: string;
-  }> =
-    await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
+  }> = await prisma.$queryRaw`SELECT tablename from pg_tables where schemaname = 'public';`;
```
Comment (overbit) [inline packages/amplication-prisma-db/prisma/clean.ts:41]: Undo this in order to fix the ci pipeline.
Locally you'd need to remove your node_modules and run npm i

**Review by overbit — CHANGES_REQUESTED** [review]

File: packages/amplication-data-service-generator/package.json [hunk 5]
```diff
@@ -5,9 +5,11 @@
   "main": "./main.js",
   "scripts": {},
   "dependencies": {
+    "@amplication/code-gen-types": "^1.1.7",
```
Comment (abrl91) [inline packages/amplication-data-service-generator/package.json:8]: This one shouldn't be here

File: helm/charts/dsg-runner-infra/templates/sensor.yaml (hunk 1 above)
Comment (abrl91) [reply helm/charts/dsg-runner-infra/templates/sensor.yaml:34]: I think this one is for development purposes and should be reverted. Make sure when you merge this PR this value represents the last image sha of the dsg

**Review by yuval-hazaz — APPROVED** [review]

**Review by yuval-hazaz — APPROVED** [review]

**Pull request merged**